pysat/instruments/templates/template_instrument.py

```diff
@@ -181,25 +178,25 @@ def preprocess(self):
 
 
 # Required function
-def list_files(tag=None, inst_id=None, data_path=None, format_str=None):
+def list_files(tag='', inst_id='', data_path=None, format_str=None):
```

Comment:
The `data_path` default is inconsistent with the description.  Should this be `''` rather than `None`?